notebooks/.ipynb_checkpoints/Untitled-checkpoint.ipynb

```diff
@@ -0,0 +1,6 @@
+{
+ "cells": [],
+ "metadata": {},
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```

Comment:
Add this file in ```.gitignore``` as we don't want each other's notebook checkpoint files.

Reply:
implemented